### ISSUE TITLE
In forum topic image lightbox scrolls up to the top of the page

### ISSUE THREAD

**Issue by ARMBouhali** [hakkens/davehakkens#69]
In forum topics .. When an picture thumbnail in a topic/reply is clicked, the screen becomes darker, the picture lightbox appears but the page jumps directly to top.

Expected behaviour : display the picture without scrolling

Possible reasons : picture src attributes look ok but the lightbox script triggers this behaviour as if an internal link (href="#") is clicked.

**Comment by MaloRchrd** [comment]
Found the answer : 
Just comment out the overflow hidden of the css class `.fancybox-lock` in file [jquery.fancybox.css](https://github.com/hakkens/davehakkens/blob/master/js/vendor/fancybox/jquery.fancybox.css#L168) line 168 

 like this and it will work 👌

```
.fancybox-lock {
    /*overflow: hidden !important;*/
    width: auto;
}
```

**Comment by davehakkens** [comment]
Awesome! :D fixed now in 2.3
